### PR TITLE
SBT-70 Tasks and Requests templates + tests:

### PR DESCRIPTION
**Pull request by quantori-cherednichenko**

### CODE REVIEW

File: src/signals_notebook/entities/tables/cell.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ class Config:
 class CellContent(GenericModel, Generic[CellContentType]):
     value: CellContentType
     values: Optional[List[CellContentType]] = None
-    type: Optional[EntityType] = None
+    type: Optional[Union[EntityType, ObjectType]] = None
```
Comment (quantori-cherednichenko) [inline src/signals_notebook/entities/tables/cell.py:98]: Andrey's bugfix will be merged.

File: src/signals_notebook/entities/todo_list/todo_list.py [hunk 2]
```diff
@@ -49,7 +51,10 @@ def _reload_tasks(self) -> None:
         self._tasks_by_id = {}
         file = self.get_content()
         content = file.content.decode('utf-8')
-        dict_content = json.loads(content)
```
Comment (quantori-cherednichenko) [inline src/signals_notebook/entities/todo_list/todo_list.py:52]: json cannot load None

File: src/signals_notebook/entities/tables/cell.py (hunk 1 above)
Comment (quantori-shishkin) [reply src/signals_notebook/entities/tables/cell.py:98]: type: Optional[Union[EntityType, MaterialType]] = None

File: src/signals_notebook/entities/request.py [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+                type=cls._get_entity_type(),
+                attributes=_Attributes(
+                    name=name,
+                    description=description,
+                ),
+                relationships=relationships,
+            )
+        )
+
+        log.debug('Creating Notebook for: %s', cls.__name__)
```
Comment (quantori-shishkin) [inline src/signals_notebook/entities/request.py:91]:
```suggestion
        log.debug('Creating Request Container for: %s', cls.__name__)
```

File: src/signals_notebook/entities/request.py [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+        log.info('Html template for %s:%s has been rendered.', self.__class__.__name__, self.eid)
+
+        return template.render(data=data)
+
+    @classmethod
+    def load(cls, path: str, fs_handler: FSHandler, notebook: Notebook) -> None:
+        from signals_notebook.item_mapper import ItemMapper
+
+        metadata = json.loads(fs_handler.read(fs_handler.join_path(path, 'metadata.json')))
+        experiment = cls.create(
```
Comment (quantori-shishkin) [inline src/signals_notebook/entities/request.py:121]:
```suggestion
        request_container= cls.create(
```

File: src/signals_notebook/entities/todo_list/task.py [hunk 5]
```diff
@@ -114,3 +116,76 @@ def save(self, force: bool = True) -> None:
...
+        """Load Task entity
+
+        Args:
+            path: content path
+            fs_handler: FSHandler
+            parent: Container where load Task entity
+
+        Returns:
+
+        """
```
Comment (quantori-shishkin) [inline src/signals_notebook/entities/todo_list/task.py:157]: ?????

File: src/signals_notebook/entities/todo_list/task.py [hunk 6]
```diff
@@ -114,3 +116,76 @@ def save(self, force: bool = True) -> None:
...
+                metadata = {
+                    'properties': [item.name for item in properties if item.name],
+                    **{k: v for k, v in template.dict().items() if k in ('name', 'description', 'eid')},
+                }
+                fs_handler.write(
+                    fs_handler.join_path(base_path, 'templates', entity_type, f'metadata_{template.name}.json'),
+                    json.dumps(metadata),
+                )
+        except TypeError:
+            pass
```
Comment (quantori-shishkin) [inline src/signals_notebook/entities/todo_list/task.py:191]: maybe add message to logger?

**Review by quantori-pokidovea — APPROVED** [review]

**Review by quantori-shishkin — APPROVED** [review]

**Pull request merged**